src/game_api/script/lua_vm.cpp

```diff
@@ -885,6 +894,8 @@ end
         return sol::nullopt;
     };
 
+    lua.create_named_table("BUTTONS", "NONE", 0, "JUMP", 1, "WHIP", 2, "BOMB", 4, "ROPE", 8, "RUN", 16, "DOOR", 32, "MENU", 64, "JOURNAL", 128, "LEFT", 256, "RIGHT", 512, "UP", 1024, "DOWN", 2048);
```

Comment:
This should probably be called `INPUT` to mirror the usage with the functions named `*_input` and avoid confusion with the enum `BUTTON`

Reply:
sure, but INPUT is already define by some windows lib, so can't make it a type for the `*_input` functions :/

Reply:
You can certainly make it a lua enum though